wasm/src/lib.rs

```diff
@@ -0,0 +1,23 @@
+extern crate console_error_panic_hook;
+extern crate nixpkgs_fmt;
+extern crate wasm_bindgen;
+extern crate wee_alloc;
```

Comment:
In Rust 2018 (what we use) there's no need to write `extern crate` anymore. Just deleting these lines should work.

Reply:
Thanks, didn't know!